### ISSUE TITLE
create new atlas for graph construction

### ISSUE THREAD

**Issue by zhangyu2ustc** [SIMEXP/fmri_predict#16]
the current MMP atlas only includes 360 regions on cortical surface. We need more subregions to improve the scale of mapping.

One idea is to break down each region of MMP atlas in order to form a new atlas of around 1000 regions

possible solution:
  sklearn.clustering using geometric distance matrix as similarity

**Comment by zhangyu2ustc** [comment]
multi-scale atlas to try:
Resting state fMRI data from 1489 subjects were registered using surface-based alignment. A gradient weighted markov random field approach was employed to identify cortical parcels ranging from 100 to 1000 parcels. 
The parcellations were first computed in Freesurfer fsaverage6 space and sampled to fsaverage5 and fsaverage space. The parcellations were also projected to *HCP fslr32k* and FSL MNI space. 

repo:
https://github.com/ThomasYeoLab/CBIG/tree/master/stable_projects/brain_parcellation/Schaefer2018_LocalGlobal


Here we will test the GCN with two resolution: 400 and 1k
the 400 parcels are similar to MMP_atlas, and we expect a similar performance
the 1000 parcels will bring higher acc of prediction model

to generate the new adj_matrix based on geometric distance:
'''
wb_command -cifti-label-adjacency Schaefer2018_1000Parcels_17Networks_order.dlabel.nii IBIG1k_adjacency_mat_white.pconn.nii -left-surface S1200.L.white_MSMAll.32k_fs_LR.surf.gii -right-surface S1200.R.white_MSMAll.32k_fs_LR.surf.gii
'''

**Comment by lunebellec** [comment]
Why not MIST?

**Comment by zhangyu2ustc** [comment]
I am currently using the surface-based CIFTI data for task fMRI. I found that the projection from volume to surface space is a bit of weird.  I have talked to Sebastian about this. He suggested that this multi-scale cortical parcellation might be a safer choice.

**Comment by zhangyu2ustc** [comment]
Using the CBIG1000 parcellation which includes 1000 parcels on the cortical surface, we found a similar performance on MOTOR task prediction:

channel        MMP       CBIG1k
17 (12s)      0.9836     0.9866
1 (0.72s)     0.7856     0.7820

More tests will  be done for ALL tasks and two scales (1k vs 400)
